client/lv_utils.py

```diff
@@ -292,10 +292,8 @@ def lv_create(vg_name, lv_name, lv_size, force_flag=True):
 
     if not vg_check(vg_name):
         raise error.TestError("Volume group could not be found")
-    if lv_check(vg_name, lv_name) and not force_flag:
+    if lv_check(vg_name, lv_name):
         raise error.TestError("Logical volume already exists")
-    elif lv_check(vg_name, lv_name) and force_flag:
-        lv_remove(vg_name, lv_name)
```

Comment:
Here, I'd say the commit message could be improved to 'Simplify lv_create behavior' instead of 'Make all lv operations more atomic'.